src/components/Theme/style.css

```diff
@@ -14,12 +31,18 @@
   font-size: 30px;
   line-height: 40px;
   text-align: center;
-  margin: 1rem 0rem;
+  margin: 4rem @large-text-margin 1rem;
```

Comment:
This affects the margin between the icon and the title on the `complete` screen. FYI I've fixed it as part of [this PR] (https://github.com/onfido/onfido-sdk-ui/pull/253)